src/vivarium/framework/event.py

```diff
@@ -169,7 +150,7 @@ def setup(self, builder):
         self.clock = builder.time.clock()
         self.step_size = builder.time.step_size()
 
-    def get_emitter(self, name: str) -> Callable:
+    def get_emitter(self, name: str) -> Callable[[pd.Index, Optional[Dict]], Event]:
```

Comment:
why include Event in the type annotation for the callable arguments to emit? Doesn't emit only take an index and user_data now?

Reply:
@kfwilson The event is the return type of the callable

Reply:
i think it is return type of the callable